include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -4273,7 +4275,8 @@ namespace internal
         }
       else if (fe_degree > -1 &&
                subface_index >= GeometryInfo<dim>::max_children_per_cell &&
-               shape_info.element_type <= MatrixFreeFunctions::tensor_symmetric)
+               shape_info.element_type <=
+                 MatrixFreeFunctions::tensor_symmetric_no_collocation)
         FEFaceEvaluationImpl<true,
```

Comment:
Isn't this branch also using collocation derivatives: https://github.com/dealii/dealii/blob/2d6e494f91682cc2f737442cff989365f40858ea/include/deal.II/matrix_free/evaluation_kernels.h#L2533-L2534
Hence, I believe we need to use `tensor_symmetric` here and in the other face function below.